OWNERS_ALIASES

```diff
@@ -60,6 +60,13 @@ aliases:
     - alexbrand
     # glo-pena
     - electrocucaracha
+  sig-docs-fa-owners: # Admins for Persian content
+    - sattarfeizollahibarough
+    - kasramp
+  sig-docs-fa-reviews: # PR reviews for Persian content
+    - sattarfeizollahibarough
+    - kasramp
+    - FaezeAzimi
```

Comment:
Excellent, thanks for adding this. Two things:

1. Alphabetize each block
2. Add the same alphabetized team blocks to https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L74.

Reply:
The requested changes have been made.
https://github.com/kubernetes/org/pull/2051